index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://sad-haibt-42112c.netlify.app/styles/main.css"
+    />
+    <link rel="stylesheet" href="/style.css" />
+    <title>Home | Sanguine store</title>
+  </head>
+  <body>
+    <nav class="navigation">
+      <div class="nav-logo">SANGUINE store</div>
+      <label for="search" class="text-size-sm">
+        <input
+          type="search"
+          class="search-box padding-md border-rad-md"
+          name="search"
+          id="search"
+          placeholder="What are you looking for?"
+        />
```

Comment:
use some border for the search container or some different background color, rest looks good